backend/docs/guide/create-feature-x.md

```diff
@@ -0,0 +1,105 @@
+# Guide on creating a feature in Infisical backend
+
+Let's say you want to implement a new feature, call it feature-x for name sake. These are steps to follow.
```

Comment:
It would be best if you can make a actual simple feature with code. This way, we can follow it along like a tutorial. 

Instead of `feature-x`, maybe you can make a `todo-list` feature? This way, the reader will get to know how to make database schema, routes, CRUD APIs, generate service, etc. 

To top it off, maybe you can also record a nice little video of this in action? For video you can use loom or this is also really nice https://www.screen.studio/download. 

The project structure break down is great.